modules/admin-top-bar/module.php

```diff
@@ -41,6 +41,8 @@ private function enqueue_scripts() {
 
 		wp_enqueue_style( 'elementor-admin-top-bar', $this->get_css_assets_url( 'admin-top-bar', null, 'default', true ), [], ELEMENTOR_VERSION );
 
+		do_action( 'elementor/admin_top_bar/before_enqueue_scripts', $this );
```

Comment:
If you add `before_enqueue_scripts` hook, maybe add an additional `after_enqueue_scripts`.

Reply:
I think when we will need it, we will add it. No before :)